inmem_store.go

```diff
@@ -106,7 +106,11 @@ func (i *InmemStore) Set(key []byte, val []byte) error {
 func (i *InmemStore) Get(key []byte) ([]byte, error) {
 	i.l.RLock()
 	defer i.l.RUnlock()
-	return i.kv[string(key)], nil
+	val := i.kv[string(key)]
+        if val == nil {
```

Comment:
Please run `go fmt` on this to get the indentation correct. 

Also I think your travis test failures are due to not importing the "errors" package.